widgetry/src/style/mod.rs

```diff
@@ -22,6 +22,7 @@ pub struct Style {
 pub struct ButtonStyle {
     pub fg: Color,
     pub fg_disabled: Color,
+    pub fg_hotkey: Color,
```

Comment:
I suspect we might want to remove `hotkey_color` and clean up some other places using this manually. I'll do an audit in a bit.

Reply:
Maybe so - it's kind of a stopgap. I think the current fg color looks good in the bottom HUD, so didn't want to change it there.

Even though it's  somewhat redundant with the tooltip display, they are kind of nice, but it's not clear to me when to use a hotkey button, since we have all kinds of text buttons that have hotkeys that *aren't* rendered as hotkey buttons. I'm not sure they're pulling their weight. It's probably controversial, but I'd actually favor simplifying and getting rid of the "hotkey buttons" all together and uniformly relying on the pop-ups to communicate hotkey discoverability.

Reply:
(That's my medium term position, in the short term I think it's fine to keep them around, which is why I tweaked the contrast here)

Reply:
>  I think the current fg color looks good in the bottom HUD, so didn't want to change it there.
![Screenshot from 2021-01-24 10-49-05](https://user-images.githubusercontent.com/1664407/105640378-89e82400-5e32-11eb-88d1-14bf90de2066.png)
Yeah, I guess green actually does show up nicely there too.

> I'd actually favor simplifying and getting rid of the "hotkey buttons" all together

I'm in agreement. The new info panel designs use custom icons for actions and don't have the hotkey at all. The layer panel is the other main place using them. I think they're probably leftover from the original UI, which was following Dungeon Crawl / Dwarf Fortress as inspiration and emphasizing mostly keyboard control: https://www.yuwen-li.com/images/abstreet/problem.png